tests/test_utils.py

```diff
@@ -35,6 +36,85 @@ def test_buildconfig_update():
     assert x == {'a': 'A', 'strategy': {'b1': 'newB1', 'b3': 'B3', 'b11': {}}, 'c': 'C', 'd': 'D'}
 
 
+def has_ist(buildconfig):
```

Comment:
Is Generic the only anticipated Other Trigger? Or would Generic return anything but ImageChange?